home/home_2.1/main.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+
+let FILENAME = "user_stats.txt"
+
+func readStats() -> [String: Int] {
+    var stats = [String: Int]()
+    if let fileContents = try? String(contentsOfFile: FILENAME) {
+        let lines = fileContents.split(separator: "\n")
+        for line in lines {
+            let parts = line.split(separator: " ")
```

Comment:
We can try to use extension of class string to parse string into expected stat values - for relevant syntax overview use for example https://www.hackingwithswift.com/quick-start/beginners/how-to-create-and-use-extensions